### ISSUE TITLE
videojs flash not working on edge.

### ISSUE THREAD

**Issue by dovelive** [videojs/videojs-flash#98]
## Description
Here is the example url for videojs flash player.
http://163.44.173.34/flash.html

And here is the code of that file.
All packages are newest ones.
```
<!doctype html>
<html>
  <head>
    <link href="//vjs.zencdn.net/7.2.4/video-js.css" rel="stylesheet">

  </head>

  <body>
    <video id=example-video width=600 height=300 class="video-js vjs-default-skin" controls></video>
    <script src="//vjs.zencdn.net/7.2.4/video.js"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/videojs-flash/2.1.2/videojs-flash.min.js"></script>
    <script src="https://cdn.jsdelivr.net/npm/@brightcove/videojs-flashls-source-handler@1.4.5/dist/videojs-flashls-source-handler.cjs.min.js"></script>

    <script>
    var player = videojs('example-video');

    player.ready(function() {
      player.src({
        src: "rtmpt://vod.videog.jp/vod/_definst_/&mp4:storage1/86/61/EgJM-KrjEHPm/hhd.mp4",
        type: 'rtmp/mp4',
      });

      player.play();
    });
    </script>
  </body>
</html>
```

It works fine on other browsers except the Edge.

![image](https://user-images.githubusercontent.com/19376598/47559444-0df78c80-d948-11e8-877e-8955692ebd4d.png)

So any help would be really appreciate.

**Comment by gkatsev** [comment]
You don't want to use the cjs or es files in the browser. You should switch to either https://cdn.jsdelivr.net/npm/@brightcove/videojs-flashls-source-handler@1.4.5/dist/videojs-flashls-source-handler.js or https://cdn.jsdelivr.net/npm/@brightcove/videojs-flashls-source-handler@1.4.5/dist/videojs-flashls-source-handler.min.js

**Comment by dovelive** [comment]
@gkatsev 
Thank you for contacting.
I modified the source code and uploaded to sample url.
However that does not solve the problem.
Would you like to check it once more?

**Comment by mister-ben** [comment]
Like many modern browsers Flash isn't easy detectable in Edge. I think we'd need to be more aggressive in `isSupported()` and skip the version check as for Safari and Chrome. Then you'd get the browser's click-to-play, _if_ it is installed and enabled. 

https://github.com/videojs/videojs-flash/blob/master/src/plugin.js#L821

**Comment by gkatsev** [comment]
Should be available now thanks to #111. We'll release a minor soon.

**Comment by gkatsev** [comment]
2.2.0 is out now with this.
